spec/app/domain/authentication/authn-oidc/authn_oidc_spec.rb

```diff
@@ -140,6 +143,7 @@
             enabled_authenticators: oidc_authenticator_name,
             token_factory: token_factory,
             validate_security: mocked_security_validator,
+            validate_account_exists: mocked_account_validator,
```

Comment:
how here do we mock here that an acc has an user that is an admin?

Reply:
You can see it here: https://github.com/cyberark/conjur/blob/master/spec/support/oidc_spec_helper.rb

for each test, the `mocked_account_validator` returns true when called upon (we don't actually check that there is an account, we just mock the validation success). additionally, in [line 117](https://github.com/cyberark/conjur/blob/master/spec/support/oidc_spec_helper.rb#L117) we have a test that verifies that if the account validation fails then the authenticator fails with the same error.

Reply:
טיל נפץ @nessiLahav